Documentation/workflow/UsingYourBuild.md

```diff
@@ -3,7 +3,7 @@
 
 We assume that you have successfully built CoreCLR repository and thus have files of the form
 ```
-    bin\Product\<OS>.<arch>.<flavor>\.nuget\pkg\Microsoft.NETCore.Runtime.CoreCLR.<version>.nupkg
```

Comment:
This can refer to ` bin\Product\<OS>.<arch>.<flavor>\`. No need to refer to nuget package.